components/shared/LanguagePill/LanguagePill.tsx

```diff
@@ -1,4 +1,5 @@
 import Image, { StaticImageData } from "next/image";
+import SwiftIcon from "img/icons/interests/swift.svg";
```

Comment:
The two icons I added are the same. Not sure what we do differently with our thumbnails since it's an SVG and be resized.